_data/cryptocurrencies.yml

```diff
@@ -415,6 +415,15 @@ websites:
       - totp
     doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
 
+  - name: STEX
+    url: https://stex.com
+    img: stex.png
+    tfa:
+      - totp
+      - email
+      - u2f
+    doc: https://help.stex.com/en/articles/3486981
```

Comment:
`https://help.stex.com/en/articles/1657681`?

Reply:
https://help.stex.com/en/articles/3486981 also mentions "Web Authentication with Security Key" a bit further down, so I think this is the better doc page.